manifests/init.pp

```diff
@@ -6,19 +6,33 @@
 #
 #
 class kmod (
-  Hash $list_of_aliases    = {},
-  Hash $list_of_blacklists = {},
-  Hash $list_of_installs   = {},
-  Hash $list_of_loads      = {},
-  Hash $list_of_options    = {},
+  Hash             $list_of_aliases,
```

Comment:
please add the default values here and not as data-in-modules. that enabled puppet-strings to pick it up and add it to the REFERENCE.md.